README.md

```diff
@@ -19,3 +19,23 @@ This repository is a monorepo of the various components required for Chainlink o
 
     # Then run
     asdf install
+
+# Running e2e tests
```

Comment:
Going forward I'd prefer all long-form docs to go under `./docs` subfolder, markdown files in the appropriate section.

Do you mind moving those to `./docs` and actually linking to them from a README. Ideally a README from the `tests/e2e` subfolder.

Reply:
Updated to this formatting